Dockerfile.ui

```diff
@@ -1,4 +1,4 @@
-FROM node:lts-alpine AS cvat-ui
+FROM node:erbium-alpine AS cvat-ui
```

Comment:
Could you use `12-alpine` tag instead of `erbium-alpine`? I'm not familiar with node release codenames, so it took me quite a while to figure it out.

Reply:
done.